<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon Card Game &lt; TCG &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseCharmander">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-image:url(&quot;images/pokeball.png&quot;);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseBulbasaur">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-image:url(&quot;images/pokeball.png&quot;);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseSquirtle">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-image:url(&quot;images/pokeball.png&quot;);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
